dbt_copilot_helper/commands/conduit.py

```diff
@@ -277,34 +276,52 @@ def start_conduit(app: str, env: str, addon_type: str, addon_name: str = None):
 
 
 @click.command(cls=ClickDocOptCommand)
-@click.argument("addon_type", type=click.Choice(CONDUIT_ADDON_TYPES))
 @click.option("--app", help="AWS application name", required=True)
 @click.option("--env", help="AWS environment name", required=True)
 @click.option("--addon-name", help="Name of custom addon", required=True)
-def conduit(addon_type: str, app: str, env: str, addon_name: str):
+def conduit(app: str, env: str, addon_name: str):
     """Create a conduit connection to an addon."""
     check_copilot_helper_version_needs_update()
 
-    try:
-        start_conduit(app, env, addon_type, addon_name)
-    except InvalidAddonTypeConduitError:
+    with open("addons.yml", "r") as fd:
```

Comment:
I'm not sure about this, it now requires the command be run from the deploy repo...

Maybe an alternative is to use the parameter store / secrets manager paths as they include the addon name.

Alternatively, as per the pipeline setup [here](https://github.com/uktrade/copilot-tools/blob/main/dbt_copilot_helper/templates/pipelines/codebase/overrides/stack.ts#L447) you could upload the addons file to the environment and use that to [load data](https://github.com/uktrade/copilot-tools/blob/main/dbt_copilot_helper/commands/codebase.py#L101) for the conduit command.

The upside to this is that you could also add a `copilot conduit list --app <app> --env <env>` command or similar.

Reply:
Didn't really think about that command not being tied to the deploy repo 🫤

Getting the config from a parameter is a nicer solution anyway, I'll add a separate ticket for a `conduit list` command.